pkg/controller/v1beta1/inferenceservice/reconcilers/ingress/ingress_reconciler.go

```diff
@@ -293,6 +293,8 @@ func createIngress(isvc *v1beta1.InferenceService, config *v1beta1.IngressConfig
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      isvc.Name,
 			Namespace: isvc.Namespace,
+			Annotations: isvc.Annotations,
+			Labels: isvc.Labels,
```

Comment:
Annotation like `kubectl.kubernetes.io/last-applied-configuration` and `creationTimestamp` shouldn't be passed down.